clj-refactor.el

```diff
@@ -2208,22 +2239,27 @@ See: https://github.com/clojure-emacs/clj-refactor.el/wiki/cljr-add-project-depe
     (cljr--add-project-dependency lib-name version)))
 
 ;;;###autoload
-(defun cljr-update-project-dependency ()
+(defun cljr-update-project-dependency (&optional version)
   "Update the version of the dependency at point."
   (interactive)
   (cljr--ensure-op-supported "artifact-list")
-  (unless (cljr--looking-at-dependency-vector-p)
+  (unless (cljr--looking-at-dependency-p)
     (user-error "Place cursor in front of dependency vector to update."))
```

Comment:
This error message might be slightly out of date now, as the edn based deps aren't vectors

Reply:
Would "Place cursor in front of dependency to update." be ok in this case?